### PR TITLE
Create new page of image analysis showcase

### LINKED ISSUE #328: New showcase page on image analysis tool
**What topic do you wish to add?**
Image analysis tool that was developed under the ISIDORe project at Euro-BioImaging

**Resources**
https://docs.google.com/document/d/1pguyVYiTt_FkuHCROM8TDCRB_-k3oh5zGmPt48hDs4w/edit?usp=sharing

### PR DESCRIPTION
**Pull request by lauportell**
Closes #328

### CODE REVIEW

File: showcase/image_analysis.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Resources
+
+1: Guedj, E et al. “18F-FDG brain PET hypometabolism in patients with long COVID.” European journal of nuclear medicine and molecular imaging vol. 48,9 (2021): 2823-2833. doi:10.1007/s00259-021-05215-4
+2: https://www.medrxiv.org/content/10.1101/2022.06.02.22275916v1 
+
+News-article: https://www.eurobioimaging.eu/news/an-open-source-image-analysis-tool-for-studying-neuroinflammation-linked-to-covid-19/
+
+MBAT on Github: https://github.com/Turku-BioImaging/mouse-brain-alignment-tool
```
Comment (bedroesb) [inline showcase/image_analysis.md:44]: Since MBAT is linked in the resource table, this is redundant imo

File: showcase/image_analysis.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+The current iteration of MBAT is only applicable to ARG images of mouse brain, as it relies on the Mouse Brain Atlas. However, by incorporating additional standardized brain atlases, such as those for the rat animal model, it could be made more widely applicable to other brain analyses. Currently, it is only compatible with ARG scans; however, with minor modifications, ARG images set up on whole slides could be accommodated. With a few minor modifications, the tool can also be applied to a range of different image file types. In essence, the tool is useful for any application that aims to measure signals in coronal slices of the brain, whether through autoradiography or alternative staining methods, including fluorescent staining. Thus, the tool can greatly facilitate neuroinflammation research, whether induced through infectious diseases or other causes.
+
+## Resources
```
Comment (bedroesb) [inline showcase/image_analysis.md:37]: I have a feeling with should impose a style guide here, now it looks a bit like "Miscellaneous section" :)

**Comment by bedroesb** [comment]
General comment:

- The file is called `image_analysis`
- The Title is: `Measuring COVID-19 induced neuroinflammation -  an automated pipeline for analysing brain autoradiography images`
- the page id is `image_analysis_isidore`
- The side bar is most of the time the same is the title, so also: `Measuring COVID-19 induced neuroinflammation -  an automated pipeline for analysing brain autoradiography images`

I feel like we should homogenize these and make the title shorter

image

**Comment by bedroesb** [comment]
More information about "Isabel Kemmer" should still be added to the CONTRIBUTORS file

**Comment by bedroesb** [comment]
@EvaGarciaAlvarez I did a more general review but did not really read the content :)

File: showcase/image-analysis.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+affiliations: [European Union, Euro-BioImaging, ISIDORe]
+page_id: image_analysis_isidore
+
+---
+
+## Introduction 
+
+Coronavirus disease 2019 (Covid-19) has been linked to a range of neurological conditions, including neuroinflammation and impaired energy metabolism in the brain. In the field of neuroscience, a variety of tracers are frequently employed to investigate diverse aspects of brain health and dysfunction. Indeed, recent articles have demonstrated the presence of pronounced inflammatory processes in the brains of long COVID patients, by the use of inflammatory and metabolic tracers in positron emission tomography (PET) imaging (1, 2) and Autoradiography (ARG).
+This showcase details the development, of an open-source, user-friendly, automated analysis pipeline for aligning and processing autoradiography images from mouse brain. Funded by the [ISIDORe project](https://isidore-project.eu/), his so called {% tool "mbat" %} facilitates enhanced efficiency, accuracy, and reproducibility in the registration and analysis of images in studies aiming to elucidate the characteristics and underlying mechanisms of neuroinflammation due to infection or other causes.
```
Comment (EvaGarciaAlvarez) [inline showcase/image-analysis.md:13]: missing "t": "t"his so called

File: showcase/image-analysis.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+
+## What is the showcase?
+
+{% include image.html file="/MBAT_scheme.png" caption="Figure 1. Development of an open source, semi-automated processing pipeline for aligning autoradiography images from mouse brain tissue to study infection-induced neuroinflammation. The python-based pipeline takes input from data generated through Euro-BioImaging and is itself developed by funding from the ISIDORe project and allows users to align the images in a Napari-based user interface to the Allen Brain Atlas. The full code is open source available on GitHub." alt="Image with a central symbolic computer screen with the program MBAT. Arrows flowing in and out of the screen denoting inputs (data generated trough Euro-Bioimaging, Funding from ISIDORe project, Tool development wiht Python and Napari) and outputs (Analyzed data and Code deposition on GitHub) of the software." %}
```
Comment (EvaGarciaAlvarez) [inline showcase/image-analysis.md:24]: - references needed for: Napari-based user interface to the Allen Brain Atlas.
- typo: data generated *trough* Euro-Bioimaging

**Review by EvaGarciaAlvarez — DISMISSED** [review]
Looks good! I only have a couple of comments (mainly typos) to address before publishing the page.

**Comment by lauportell** [comment]
Add ISIDORe logo in Affiliations

File: showcase/image-analysis.md (hunk 3 above)
Comment (lauportell) [reply showcase/image-analysis.md]: done

File: showcase/image-analysis.md (hunk 4 above)
Comment (lauportell) [reply showcase/image-analysis.md]: done

File: showcase/image_analysis.md (hunk 1 above)
Comment (lauportell) [reply showcase/image_analysis.md]: done

**Comment by lauportell** [comment]
> More information about "Isabel Kemmer" should still be added to the CONTRIBUTORS file

done!

**Comment by lauportell** [comment]
> General comment:
> 
> * The file is called `image_analysis`
> * The Title is: `Measuring COVID-19 induced neuroinflammation -  an automated pipeline for analysing brain autoradiography images`
> * the page id is `image_analysis_isidore`
> * The side bar is most of the time the same is the title, so also: `Measuring COVID-19 induced neuroinflammation -  an automated pipeline for analysing brain autoradiography images`
> 
> I feel like we should homogenize these and make the title shorter
> 
> image

done!

**Comment by lauportell** [comment]
> Add ISIDORe logo in Affiliations

done!

**Comment by lauportell** [comment]
I think all changes have been addressed, let me know if there's anything else needed

**Comment by bedroesb** [comment]
Looks good now!

**Review by bedroesb — APPROVED** [review]

**Review by bedroesb — APPROVED** [review]

**Pull request merged**